service/src/test/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelperTest.java

```diff
@@ -22,6 +22,7 @@
 import static uk.gov.hmcts.reform.fpl.utils.CaseDataGeneratorHelper.createHearingBookingsFromInitialDate;
 import static uk.gov.hmcts.reform.fpl.utils.CaseDataGeneratorHelper.createRespondents;
 import static uk.gov.hmcts.reform.fpl.utils.DateFormatterHelper.formatLocalDateTimeBaseUsingFormat;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.unwrapElements;
 import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.formatCaseUrl;
 
 @ExtendWith(SpringExtension.class)
```

Comment:
Should't need any spring configuration with this. It is no longer managed by spring so any autowiring is redundant

Reply:
good spot removed :)